FFMpegCore/FFMpeg/FFMpegArgumentOptions.cs

```diff
@@ -52,6 +52,7 @@ public FFMpegArgumentOptions WithVideoFilters(Action<VideoFilterOptions> videoFi
         public FFMpegArgumentOptions Seek(TimeSpan? seekTo) => WithArgument(new SeekArgument(seekTo));
         public FFMpegArgumentOptions Loop(int times) => WithArgument(new LoopArgument(times));
         public FFMpegArgumentOptions OverwriteExisting() => WithArgument(new OverwriteArgument());
+        public FFMpegArgumentOptions SelectStream(int index) => WithArgument(new MapStreamArgument(index));
```

Comment:
Add second argument `inputFileIndex` for specifying the input file index, in case multiple input file are provided. 
It should probably just default to 0?